boards/opentitan/src/tests/otbn.rs

```diff
@@ -81,10 +82,26 @@ extern "C" {
     static _eapps: u8;
 }
 
+/// Static init an OtbnTestCallback, with
+/// respective buffers allocated
+/// Note: static_init!() returns an &'static mut reference.
+unsafe fn static_init_test_cb() -> &'static OtbnTestCallback {
+    let output_buf = static_init!([u8; 1024], [0; 1024]);
+    let temp_buf = static_init!([u8; 4], [0; 4]);
+
+    static_init!(
+        OtbnTestCallback,
+        OtbnTestCallback::new(output_buf, temp_buf)
+    )
+}
+
 #[test_case]
 fn otbn_run_rsa_binary() {
     let perf = unsafe { PERIPHERALS.unwrap() };
     let otbn = &perf.otbn;
+    let cb = unsafe { static_init_test_cb() };
+    let output = cb.output_buf.take().unwrap();
```

Comment:
You need to save this again in `op_done()`

Reply:
Thanks, I have made these changes.